templates/layouts/mojolicious.html.ep

```diff
@@ -8,8 +8,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title><%= title || 'Mojolicious - Perl real-time web framework' %></title>
     %= javascript '/mojolicious/jquery/jquery.js'
-    %= javascript '/mojolicious/prettify/run_prettify.js'
-    %= stylesheet '/mojolicious/prettify-mojo-light.css'
+    %= javascript '/mojolicious/highlight.js/highlight.min.js'
+    %= javascript '/mojolicious/highlight.js/mojolicious.min.js'
+    %= javascript '/mojolicious/highlight.js/apache.min.js'
+    %= javascript '/mojolicious/highlight.js/ini.min.js'
+    %= javascript '/mojolicious/highlight.js/nginx.min.js'
+    %= javascript '/mojolicious/highlight.js/yaml.min.js'
+    %= stylesheet '/mojolicious/highlight.js/highlight-mojo-light.css'
+    <script>hljs.initHighlightingOnLoad();</script>
```

Comment:
I think it’s better to start highlighting right before </body> instead of onload()

Reply:
Just following what's on the tin: https://highlightjs.org/usage/

There's also option for customizing the load, but it still attaches to `DOMContentLoaded`; let me know if you have a better suggestion.